koku/api/metrics/views.py

```diff
@@ -36,23 +40,42 @@ class CostModelMetricsMapViewSet(mixins.ListModelMixin, viewsets.GenericViewSet)
 
     """
 
-    queryset = CostModelMetricsMap.objects.all()
     serializer_class = CostModelMetricMapSerializer
```

Comment:
Can this just be the `Serializer` class  `from rest_framework.serializers import Serializer`? as mentioned above

Reply:
ok. thanks for trying that. Would have preferred to simplify this for readability, but looks like it needs to stay as is.